panel/models/pdf.ts

```diff
@@ -0,0 +1,79 @@
+import * as p from "@bokehjs/core/properties";
+import { Markup } from "@bokehjs/models/widgets/markup";
+import { PanelMarkupView } from "./layout";
+import { htmlDecode } from "./html";
+
+export class PDFView extends PanelMarkupView {
+  model: PDF;
+
+  connect_signals(): void {
+    super.connect_signals();
+    const p = this.model.properties;
+    const { text, width, height, embed, start_page } = p;
+    this.on_change([text, width, height, embed, start_page], () => {
```

Comment:
Do you think we can somehow control the page independently of the embedded content to avoid re-rendering the PDF from scratch each time?

Reply:
I don't think it is possible with the current implementation. This is why I called it `start_page` and not `page`. If we want full control over the PDF, we need to change the implementation to use `PDF.js`.